test/e2e/propagationpolicy_test.go

```diff
@@ -442,6 +442,18 @@ var _ = ginkgo.Describe("[BasicPropagation] basic propagation testing", func() {
 					clusterDynamicClient := getClusterDynamicClient(cluster.Name)
 					gomega.Expect(clusterDynamicClient).ShouldNot(gomega.BeNil())
 
+					klog.Infof("Waiting for crd(%s) present on cluster(%s)", crd.Name, cluster.Name)
+					crdErr := wait.Poll(pollInterval, pollTimeout, func() (done bool, err error) {
+						if _, err := clusterDynamicClient.Resource(crdGVR).Get(context.TODO(), crd.Name, metav1.GetOptions{}); err != nil {
+							if errors.IsNotFound(err) {
+								return false, nil
+							}
+							return false, err
+						}
+						return true, nil
+					})
+					gomega.Expect(crdErr).ShouldNot(gomega.HaveOccurred())
+
```

Comment:
Why we need this change? Does this necessary?
@XiShanYongYe-Chang Do you know?

Reply:
Hi, I try sometimes without this e2e code change(other changes of current pr are in use normal), it run correctly. Have you run failed with this e2e case? @huiwq1990

Reply:
Not found failed case.
But I want to check cluster resources create in member cluster. If clusterBindingReconcileWorker moduel not work, the mine test code will fail. Otherwise, I will fail when deployment sync.

Reply:
Sorry, I didn't get what you mean.
If not mandatory, I'd prefer not to change E2E in this PR.  You can file another PR to fix or improve the E2E test if it not block this PR.